flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/util/DockerImageVersions.java

```diff
@@ -28,7 +28,9 @@
  */
 public class DockerImageVersions {
 
-    public static final String KAFKA = "confluentinc/cp-kafka:6.2.2";
+    public static final String KAFKA = "confluentinc/cp-kafka:7.2.2";
+
+    public static final String SCHEMA_REGISTRY = "confluentinc/cp-schema-registry:7.2.2";
```

Comment:
also needs to be added in `tools/azure-pipelines/cache_docker_images.sh`

Reply:
I'll add it. Any reason why we don't cache all images? Like RabbitMQ, Kinesalite, Minio etc?

Reply:
I don't remember. I think because in some cases we pull LATEST, so we'd invalidate the cache quite frequently.